### DISCUSSION TITLE
Backfilling using promtool and docker

### DISCUSSION THREAD

**Discussion by pietrarenner** [prometheus/prometheus#14274] [Q&A]
Hi! 

I'm using docker to run my application and I wanted to mock some data in Prometheus metrics. I found out that it is possible to do that using the command ``promtool tsdb create-blocks-from openmetrics <input file> [<output directory>]``, as described in Prometheus' documentation, but I couldn't find any tutorials on how to do that when using docker (in my case I am specifically using a docker-compose file). If anyone could point me to tutorials that show how to do that, or could explain to me how it's done I would be very grateful! 

Thank you in advance!

**Comment by dozsa** [comment]
I just did that earlier today. Just exec into the container and run promtool inside the container.

**Reply by pietrarenner** [reply]
That worked! Thank you!
